server/services/zwave/lib/utils/bindValue.js

```diff
@@ -1,27 +0,0 @@
-const { COMMAND_CLASSES, INDEXES } = require('../constants');
-
-/**
- * @description Bind value
- * @param {Object} value - Value object.
- * @param {string} productId - ProductId string.
- * @param {number} comclass - The comclass.
- * @param {number} index - The index.
- * @returns {number} Return the value adapted.
- * @example
- * const value = bindValue(6, 0x4501, 12, 1);
- */
-function bindValue(value, productId, comclass, index) {
-  // Zipato Mini Keypad RFID
-  if (
-    productId === '0x4501' &&
```

Comment:
I know that this file was not used, but it's actually something useful and we talked about it recently on the forum:

https://community.gladysassistant.com/t/integration-zipato-mini-keypad/6325/18?u=pierre-gilles

Reply:
I'll double check this.

Reply:
It was not used, but we plan to use this! @tagcashdev worked on it

Reply:
Ok, I'll bring back this code in "the new way" here

Reply:
Not sure where this was planned to be used... I supposed it was on valueChanged. The "code" is back at least. If we need to move it, we can.